### PR TITLE
extract managedResourceNames

### PR DESCRIPTION
**Pull request by maorfr**
part of https://issues.redhat.com/browse/APPSRE-4355

also adds the important `additionalProperties: false`

### CODE REVIEW

File: schemas/openshift/managed-resource-name-1.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    type: string
+  resourceNames:
+    type: array
+    items:
+      type: string
+    minItems: 1
+
+required:
+- resource
+- resourceNames
```
Comment (patjlm) [inline schemas/openshift/managed-resource-name-1.yml:23]: is `resourceNames` really required?

Comment (maorfr) [reply schemas/openshift/managed-resource-name-1.yml:23]: what would we do without it?

Comment (patjlm) [reply schemas/openshift/managed-resource-name-1.yml:23]: only limit the kind? I think we do that most of the time. Don't we?

Comment (maorfr) [reply schemas/openshift/managed-resource-name-1.yml:23]: this is `managedResourceNames`, not `managedResourceTypes`

Comment (patjlm) [reply schemas/openshift/managed-resource-name-1.yml:23]: 🤦

**Review by patjlm — APPROVED** [review]

**Pull request merged**